dev-docs/bidder-adaptor.md

```diff
@@ -271,6 +271,20 @@ Building the request will use data from several places:
 * **BidRequest Params**: Several important parameters such as first-party data, userId, GDPR, USP, and supply chain values are on the `bidderRequest` object.
 * **Prebid Config**: Publishers can set a number of config values that bid adapters should consider reading.
 
+#### Compression Support for Outgoing Requests
+
+Prebid.js core now includes support for gzip compression of bidder request payloads (This can help reduce payload size and improve performance).
+
+The following criteria must be met for a request to be compressed:
+
+* Browser supports gzip compression
```

Comment:
we should calrify that browser compatability check is conducted by prebid core/util